common/src/main/java/org/conscrypt/Conscrypt.java

```diff
@@ -59,6 +60,13 @@ public static void checkAvailability() {
     }
 
     /**
+     * Indicates whether the given object is a Conscrypt {@link Provider}.
+     */
+    public static boolean isConscrypt(Provider provider) {
+        return provider instanceof OpenSSLProvider;
```

Comment:
Isn't that logic true for the other `isConscrypt` methods as well? So they would need to be updated as well.